doc/tutorials/Content_Page.md

```diff
@@ -12,15 +12,24 @@
   * [Docker Container Setup](./Chapter_1/01-Setup_Installation.md#docker-container-setup)
```

Comment:
Maybe rename `assets/And_gate.png` to `assets/and_gate.png` to follow `snake_case`?